setup.py

```diff
@@ -92,7 +92,7 @@
             "vowpalwabbit>=8.10.0, <9.0.0",
         ],
         "nlp": [
-            "transformers[torch]==4.18",
+            "transformers[torch]",
```

Comment:
What about using a specific version here? That way, we can make sure when the user installs the [nlp] option, the code doesn't stop working when future version of transformers is released.

Reply:
Updated